styles/style.css

```diff
@@ -0,0 +1,171 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto');
+@import url('https://fonts.googleapis.com/css?family=Noto+Serif+TC');
+
+
+* {
+    font-family: 'Roboto', sans-serif;
+}
+
+.container {
```

Comment:
i like that you the entire thing in a container class and gave it a padding to acieve the spacing!